code/controllers/StaticExporter.php

```diff
@@ -47,7 +47,10 @@ public function __construct() {
 
 		if(class_exists('SiteTree')) {
 			if(!$this->config()->get('disable_sitetree_export')) {
-				$objs[] = $this->config()->export_objects;
+				$objs = $this->config()->export_objects;
+				if (!is_array($objs)) {
+					$objs = array();
```

Comment:
Pass $obj into the array so we don't loose it if someone sets it to plain string.